src/documents/jqUnit.md

```diff
@@ -0,0 +1,305 @@
+---
+title: jqUnit
+layout: default
+category: Infusion
+---
+
+Bundled with infusion is ***jqUnit***, a library modelled on the [xUnit](https://en.wikipedia.org/wiki/XUnit) API style which wraps base functionality provided by the
+popular [QUnit](https://qunitjs.com/). All of the QUnit base functionality remains available in a jqUnit fixture at the `QUnit` namespace, and you should consult the QUnit documentation
+for details of packaging markup-based fixtures and its own API semantic. However, jqUnit is a complete wrapper and it is possible and recommended to write complete
+test suites without reference to QUnit. jqUnit is based on the 1.x API of QUnit, and not the (currently unreleased) 2.x version of QUnit which is API incompatible with QUnit 1.x.
+
+As well as use in the browser, there is also a (currently unstable) node.js module, [node-jqunit](https://github.com/fluid-project/node-jqUnit) which allows the use of the same
```

Comment:
This phrase "As well as use in the browser" doesn't seem to fit with the rest of the paragraph very well. Is there a type? a word missing?